rotkehlchen/tests/api/test_balances.py

```diff
@@ -917,17 +918,19 @@ def test_blockchain_balances_refresh(rotkehlchen_api_server: 'APIServer', ethere
     )]
     vaults_patch = patch('rotkehlchen.chain.ethereum.modules.makerdao.vaults.MakerdaoVaults.get_vaults', side_effect=lambda: makerdao_vault)  # noqa: E501
 
+    a_usdc = EvmToken(identifier=A_USDC.identifier)
```

Comment:
just use A_USDC.resolve_to_token() isntead here and below to initialize it. And then keep what you did for using it as a variable